ui/apps/platform/src/Containers/Clusters/InitBundles/InitBundleForm.tsx

```diff
@@ -170,16 +169,13 @@ function InitBundleForm(): ReactElement {
                             label="Installation method for secured cluster services"
                             isRequired
                         >
-                            <Select
-                                variant="single"
+                            <SelectSingle
```

Comment:
I did notice that the options don't fill the width. I think you may need to add a `maxWidth="100%"` prop to fix that. I need to rethink how the `maxWidth` prop works. It should do that by default.

<img width="1394" height="170" alt="Image" src="https://github.com/user-attachments/assets/91b681a0-e798-40da-b30e-4c26ff4f1a93" />